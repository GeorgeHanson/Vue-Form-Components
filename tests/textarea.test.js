import Vue from 'vue'
import TextAreaComponent from '../src/Components/TextArea.vue';

describe('TextAreaComponent', () => {
    test("it generates the markup", () => {
        const renderer = require('vue-server-renderer').createRenderer();
        const vm = new Vue({
            el: document.createElement('div'),
            render: (createElement) => {
                return createElement(
                    TextAreaComponent,
                    {
                        props: {
                            name: 'name',
                            label: 'Name'
                        }
                    }
                )
            },
        });

        renderer.renderToString(vm, (err, str) => {
            expect(str).toMatchSnapshot();
        });
    });
});