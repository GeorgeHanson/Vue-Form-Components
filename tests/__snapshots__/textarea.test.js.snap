// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextAreaComponent it generates the markup 1`] = `
< div data - server - rendered = "true"
class = "form-group" > < div class = "col-sm-4" > < label class = "control-label" > Name: <!----></label> <!----></div> <div class="control-container col-sm-8"><textarea name="name" id="name" rows="5" class="form-control"></textarea> <!----></div> <div class="clearfix"></div></div>
`;
